### PR TITLE
docs: add samples from video/cloud-client/quickstart

### PR DESCRIPTION
**Pull request by danoscarmike**
Samples from https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/video/cloud-client/quickstart

### CODE REVIEW

**Comment by danoscarmike** [comment]
Moving samples from another repository (python-docs-samples)

**Review by busunkim96 — APPROVED** [review]

**Comment by danoscarmike** [comment]
@telpirion @sirtorry another one for your stamp of approval.

File: samples/quickstart/quickstart.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        'gs://cloud-samples-data/video/cat.mp4', features=features)
+    print('\nProcessing video for label annotations:')
+
+    result = operation.result(timeout=120)
+    print('\nFinished processing.')
+
+    # first result is retrieved because a single video was processed
+    segment_labels = result.annotation_results[0].segment_label_annotations
+    for i, segment_label in enumerate(segment_labels):
+        print('Video label description: {}'.format(
```
Comment (dinagraves) [inline samples/quickstart/quickstart.py:42]: f-strings are more compact and readable: 

```suggestion
        print(f'Video label description: {category_entity.description}'
```

File: samples/quickstart/quickstart.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            segment_label.entity.description))
+        for category_entity in segment_label.category_entities:
+            print('\tLabel category description: {}'.format(
```
Comment (dinagraves) [inline samples/quickstart/quickstart.py:45]:
```suggestion
            print(f'\tLabel category description: {category_entity.description}'
```

File: samples/quickstart/quickstart.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+                category_entity.description))
+
+        for i, segment in enumerate(segment_label.segments):
+            start_time = (segment.segment.start_time_offset.seconds +
+                          segment.segment.start_time_offset.nanos / 1e9)
+            end_time = (segment.segment.end_time_offset.seconds +
+                        segment.segment.end_time_offset.nanos / 1e9)
+            positions = '{}s to {}s'.format(start_time, end_time)
```
Comment (dinagraves) [inline samples/quickstart/quickstart.py:53]:
```suggestion
            positions = f'{start_time}s to {end_time}s'
```

File: samples/quickstart/quickstart.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+            confidence = segment.confidence
+            print('\tSegment {}: {}'.format(i, positions))
```
Comment (dinagraves) [inline samples/quickstart/quickstart.py:55]:
```suggestion
            print(f'\tSegment {i}: {positions}')
```

File: samples/quickstart/quickstart.py [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+            print('\tConfidence: {}'.format(confidence))
```
Comment (dinagraves) [inline samples/quickstart/quickstart.py:56]:
```suggestion
            print(f'\tConfidence: {confidence}')
```

**Review by dinagraves — COMMENTED** [review]
nit: f-strings would make these print statements more compact and readable.

File: samples/quickstart/quickstart.py (hunk 1 above)
Comment (danoscarmike) [reply samples/quickstart/quickstart.py:42]: Totally agree. Thanks for making the changes.

**Comment by telpirion** [comment]
Question: With the failing test (`beta_snippets_test.test_detect_shot_change_streaming`), is it possible to switch to a `min_passes`, `max_tries` strategy?

**Pull request closed without merging**